### PR TITLE
fix(date-time): broken style

### PR DESCRIPTION
**Pull request by jefrydco**
https://user-images.githubusercontent.com/20434351/172538633-57c2e51e-7a97-4381-9f7e-8e04729ada19.mp4

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { addDecorator } from '@storybook/react'
 import { ThemeProvider } from '../dist'
 
-import 'antd/dist/antd.css'
```
Comment (quy-truong) [inline .storybook/preview.js:4]: why we remove `antd`? 🤔

File: src/styles/less/style.less [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @root-entry-name: default;
 
-@import '~antd/lib/style/components.less';
+@import '~antd/dist/antd.less';
```
Comment (jefrydco) [inline src/styles/less/style.less:3]: @quy-truong It's not removed, it moves into the style itself.

File: src/components/atoms/DateTimePicker/DateTimePicker.stories.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { Story } from '@storybook/react'
-import dayjs from 'dayjs'
+import moment from 'moment'
```
Comment (quy-truong) [inline src/components/atoms/DateTimePicker/DateTimePicker.stories.tsx:2]: why we switch to use `moment`? 🤔

File: src/styles/less/style.less (hunk 2 above)
Comment (quy-truong) [reply src/styles/less/style.less:3]: IIRC, there is an issue with using `antd.less` directly 🤔

File: src/components/atoms/DateTimePicker/DateTimePicker.stories.tsx (hunk 3 above)
Comment (jefrydco) [reply src/components/atoms/DateTimePicker/DateTimePicker.stories.tsx:2]: @quy-truong Customize component using this function

```
const AntDatePicker = generatePicker<Dayjs>(dayjsGenerateConfig)
```

somehow doesn't pick up the global prefix configuration nor the local. We make it work first then improve later

**Review by quy-truong — APPROVED** [review]

**Pull request merged**